### PR TITLE
Combining production and spb-analysis tools code

### PR DESCRIPTION
**Pull request by morgantaschuk**

### CODE REVIEW

File: bamqc.pl [hunk 1]
```diff
@@ -52,8 +52,8 @@ =head1 AUTHOR
 
 
 use lib dirname (__FILE__);
-use bamqc;
-
+use GSI::bamqc;
+use GSI::report;
```
Comment (lheisler) [inline bamqc.pl:56]: use lib dirname (__FILE__); allows loading of libraries in the same directory of the script this is referencing them.  if bamqc in now under GSI::, then I don't think this line is needed

File: jsonToGenericRunReport.pl [hunk 2]
```diff
@@ -28,70 +28,28 @@
 use vars qw/ %opt /;
 
 use lib dirname (__FILE__);
-use bamqc;
+
```
Comment (lheisler) [inline jsonToGenericRunReport.pl]: use lib dirname (FILE); allows loading of libraries in the same directory of the script this is referencing them. if bamqc in now under GSI::, then I don't think this line is needed

**Review by lheisler — APPROVED** [review]
Made one note on no longer needing use lib dirname,

In general, I can see that most of the work here is reorganization of the code, and for the most part it's clear how this is being done.  No concerns.

File: bamqc.pl (hunk 1 above)
Comment (morgantaschuk) [reply bamqc.pl:56]: bamqc.pl is still in the base folder. bamqc.pm and report.pm are together in GSI:: though.

**Pull request merged**